garrysmod/lua/autorun/game_hl2.lua

```diff
@@ -1,79 +1,112 @@
-
-local Category = ""
-
-local function ADD_ITEM( name, class, offset, extras, classOverride )
-
-	local base = { PrintName = name, ClassName = class, Category = Category, NormalOffset = offset or 32, DropToFloor = true, Author = "VALVe" }
-	list.Set( "SpawnableEntities", classOverride or class, table.Merge( base, extras or {} ) )
-	duplicator.Allow( class )
-
+local spawnableEntities = list.GetForEdit( "SpawnableEntities" )
+local npcWeapons = list.GetForEdit( "NPCUsableWeapons" )
+local playerWeapons = list.GetForEdit( "Weapon" )
+local Allow = duplicator.Allow

```

Comment:
I can tell you right now, you do not need to do this silly stuff here.

Reply:
I don't know coding and tried to modify unknowns code and trying to fix rn.